backend/package.json

```diff
@@ -38,6 +38,7 @@
     "@fastify/static": "6.10.2",
     "@fastify/swagger": "8.8.0",
     "@fastify/swagger-ui": "1.9.3",
+    "axios": "1.5.0",
```

Comment:
Do we need axios? node 18 supports fetch

Reply:
Not resolved